src/MainWindow.vala

```diff
@@ -99,10 +104,28 @@ public class Mail.MainWindow : Gtk.ApplicationWindow {
             message_list_box.set_conversation (node);
         });
 
+        set_paned_start_grid_width ();
+        set_search_entry_width ();
+
+        paned_end.notify["position"].connect (() => set_search_entry_width ());
+
+        paned_start.notify["position"].connect (() => {
+            set_paned_start_grid_width ();
+            set_search_entry_width ();
+        });
+
         Backend.Session.get_default ().start.begin ();
     }
 
     private void on_compose_message () {
         new ComposerWindow ().show_all ();
     }
+
+    private void set_search_entry_width () {
```

Comment:
Can we change this name to something like `update_search_entry_width`? It is confusing when a method name starts with `set` but doesn't take any parameters.